test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -76,11 +76,8 @@ extern void setMongoConnectionForUnitTest(DBClientBase*);
 * registrations collection.
 *
 */
-static void prepareDatabase(void) {
-
-  /* Set database */
-  setupDatabase();
```

Comment:
I removed it, as all functions in this module now call `utInit()`, which calls `setupDatabase()`.
Actually, it was just one of many desperate tests trying to fix the problems in the unit tests.
However, the change is good and I let it stay.